src/main/liberty/config/server.xml

```diff
@@ -11,5 +11,8 @@
 	<httpEndpoint host="*" httpPort="${default.http.port}" httpsPort="${default.https.port}" id="defaultHttpEndpoint"/>
 
 	<webApplication location="CloudantSample.war" contextRoot="/"/>
+	<variable name="cloudant.apikey" value="${API_KEY}"/>
+    <variable name="cloudant.host" value="${HOST}"/>
```

Comment:
```suggestion
```
For the sample this will need to test against a container and should be available on localhost, which is the default so this doesn't need to be specified